errors/errors.js

```diff
@@ -0,0 +1,20 @@
+exports.handlePsqlErrors = (err ,req, res, next) => {
+    if (err.code){
+        res.status(400).send({msg:"Bad request"})
+    }
+    else next(err);
+    }
+    
+    
+    exports.handleCustomErrors = (err ,req, res, next) => {
+        if (err.msg){
+            res.status(err.status).send({msg: err.msg, error:err})
+        }
+        else next(err);
+        }
+    
+    exports.handleServerErrors = (err ,req, res, next) => {
+        console.log(err, " other error take note")
+        res.status(500).send({msg: "server error, real bad, please send help!!!"})
+        }
+        
```

Comment:
Really nice looking middleware functions here Gary! However, do we need these custom error and psql error handlers yet? Have we handled any errors related to this endpoint? Be careful to only include code in each push related to the task you are on. So please remove for now :)